db/controllers/articles-controllers.js

```diff
@@ -35,4 +36,19 @@ const getCommentsById = (request, response, next) => {
     })
 }
 
-module.exports = { getArticleById, getArticles, getCommentsById }
\ No newline at end of file
+const postCommentById = (request, response, next) => {
+    const id = request.params.article_id
+    const newBody = request.body
+    const promises = [fetchArticleById(id), insertCommentById(newBody, id)]
```

Comment:
the article_id is a foreign key on the comments table so it is not necessary to check that it exists because sql will throw an error if the article does not exist